### PR TITLE
Button

### PR DESCRIPTION
**Pull request by reubenjnazareth**
Button component completed.

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    border-radius: 0.4rem;
+}
+
+.btn-float {
+    background-color: var(--primary-color);
+    border: none;
+    color: white;
+    /* padding: 1rem; */
+    cursor: pointer;
+    font-size: 2rem;
```
Comment (Rockyraven) [inline components/button.css:46]: remove that unwanted code

File: button.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <header class="header">
+        <h1 class="logo">BEN-UI</h1>
+    
+    <nav class="header-nav">
+        <ul class="nav-link-ul">
+            <li class="nav-link"><a href="index.html">Home</a></li>
+            <li class="nav-link"><a href="page1.html">Documentation</a></li>
+            <li class="nav-link"><a href="#">GITHUB</a></li>
+        </ul>
+    </nav>
```
Comment (Rockyraven) [inline button.html:22]: you can update your social media link

File: button.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+      
+</div>
+
+
+
+<hr class="line">
+
+    
+</body>
```
Comment (Rockyraven) [inline button.html:101]: remove that unwanted code 
Do your code in format document

**Review by Rockyraven — APPROVED** [review]
do some minar changes
other wise your code is clean

File: button.html (hunk 2 above)
Comment (reubenjnazareth) [reply button.html:22]: done!

File: components/button.css (hunk 1 above)
Comment (reubenjnazareth) [reply components/button.css:46]: done!

**Review by chanchal16 — APPROVED** [review]
- The code looks clean.
- Good that you gave meaningful class names.
Good to merge!

**Pull request merged**